mods/ra2/weapons/missiles.yaml

```diff
@@ -240,13 +240,13 @@ HornetBombE:
 ASWBomb:
 	Inherits: HornetBomb
 	Range: 3c0
-	ValidTargets: Underwater
+	ValidTargets: Submarine
```

Comment:
This doesn't look correct since I wouldn't classify squids or dolphins as submarines.

Reply:
Renamed it and applied the fix to squids and dolphins as well.